examples/models/llama/export_llama_lib.py

```diff
@@ -887,6 +907,34 @@ def _to_edge_and_lower_llama_xnnpack(
     return builder.to_executorch(passes=additional_passes)
 
 
+def _to_edge_and_lower_llama_openvino(
```

Comment:
I dont like this as a separate function. We should aim to just leverage partitioners and use `openvino_executor_runner`